control_loop_frequency: 3 # Hz
stop_button_debounce_time: 500 # ms
niryo_studio_timeout: 4 # Seconds